client/extensions/woocommerce/app/dashboard/pre-setup-view.js

```diff
@@ -18,36 +19,75 @@ import {
 	areSettingsGeneralLoaded,
 	getStoreLocation,
 } from 'woocommerce/state/sites/settings/general/selectors';
+import BasicWidget from 'woocommerce/components/basic-widget';
 import { errorNotice } from 'state/notices/actions';
+import { getContactDetailsCache } from 'state/selectors';
 import { getCountryData, getCountries } from 'woocommerce/lib/countries';
 import { setSetStoreAddressDuringInitialSetup } from 'woocommerce/state/sites/setup-choices/actions';
 import SetupFooter from './setup-footer';
 import SetupHeader from './setup-header';
 import { doInitialSetup } from 'woocommerce/state/sites/settings/actions';
+import QueryContactDetailsCache from 'components/data/query-contact-details-cache';
 import QuerySettingsGeneral from 'woocommerce/components/query-settings-general';
 
 class PreSetupView extends Component {
 	constructor( props ) {
 		super( props );
 		this.state = {
-			address: props.address,
+			address: {},
 			isSaving: false,
 			userBeganEditing: false,
 		};
 	}
 
 	static propTypes = {
-		site: PropTypes.shape( {
-			ID: PropTypes.number.isRequired,
-		} ),
+		siteId: PropTypes.number,
```

Comment:
<3, shouldn't this be required too?

Reply:
Also `contactDetails`, `settingsGeneralLoaded`, `storeLocation`, should be added here.

Reply:
I don't usually add the mapped selectors, but it can't hurt - will do

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/19413/commits/48914387c2e937112afeb8eebee34f32a16b6ef2